troposphere/views/app.py

```diff
@@ -129,6 +129,10 @@ def _populate_template_params(request, maintenance_records, notice_t, disabled_l
     template_params['WEB_DESKTOP_INCLUDE_LINK'] = \
         settings.WEB_DESKTOP_INCLUDE_LINK
 
+    # Tentative ATMO-981 Feature Flag
+    template_params['BATCH_RESOURCE_ACTIONS'] = \
+        settings.BATCH_RESOURCE_ACTIONS
```

Comment:
This flag would provide the quick "go back to how it was before". Granted, the previous functionality is sub-optimal.